modules/data-streams/src/internalClusterTest/java/org/elasticsearch/datastreams/IngestFailureStoreMetricsIT.java

```diff
@@ -57,7 +57,7 @@
 /**
  * An integration test that verifies how different paths/scenarios affect the APM metrics for failure stores.
  */
-@ESIntegTestCase.ClusterScope(numDataNodes = 0, numClientNodes = 0, scope = ESIntegTestCase.Scope.SUITE)
+@ESIntegTestCase.ClusterScope(scope = ESIntegTestCase.Scope.TEST, numDataNodes = 1, numClientNodes = 0, supportsDedicatedMasters = false)
```

Comment:
Some tests depend on there being exactly one node in the cluster, which I solved like this.